app/controllers/posts_controller.rb

```diff
@@ -10,4 +10,32 @@ def show
     @post = @user.posts.find(params[:id])
     @comments = @post.comments
   end
+
+  def new
+    @user = User.find(params[:user_id])
+    @post = @user.posts.build
+  end
+
+  def create
+    @user = User.find(params[:user_id])
+    @post = Post.new(
+      author: @user,
+      title: params[:post][:title],
+      text: params[:post][:text],
+      comments_counter: 0,
+      likes_counter: 0
+    )
+
+    if @post.save
+      flash[:success] = 'Post created successfully!'
+      redirect_to user_posts_path(@user)
+    else
+      flash[:error] = 'Error creating the post.'
+      render 'new'
+    end
```

Comment:
- [ ] Good job creating the Post create method, note one imported thing : the post should e created on the behalf of the `current_user` which is f the currently logged-in user 👍 

```ruby
.
.
.
def create
      @post = @current_user.posts.new(post_params)
.
.
.
```